__tests__/app.test.js

```diff
@@ -34,7 +34,48 @@ describe("GET /api/topics", () => {
       .expect(404)
       .then((response) => {
         const error = response.body;
-        expect(error.msg).toBe("path not found")
-    });
+        expect(error.msg).toBe("path not found");
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with an article object, which should have all relevant properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body;
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Because we are requesting a particular article we should assert that it has the correct article_id he, not just any number.

Reply:
Thanks, I've added the correct id (1) to tthe expected return object.